src/check_list.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+"""
+    <name>
+        check_list.py
+
+    <Description>
+        Obtains a random word from /usr/share/words and checks
+        if there is a list of that. 
+
+    <Usage>
+        from check_list import check_random_list, LOLException
+        try:
+            check_random_list()
+        Except LOLException as e:
+            print("{}".format(e))
+        else:
+            print("No new list-less object for LOL")
+""" 
+import wikipedia
+import linereader
+import random
+
+DICT = "../dict.txt"
+WC_DICT = 48915
+
+class LOLException(Exception):
```

Comment:
Took me a little bit to understand the significance of this, for it is an exception, nay an error if you find an object without a list! Would you mind describing that in the docstring of this exception? i.e. raised when .... etc